pages/docs/1_query/1_getting_started/6_setup_endpoint.md

```diff
@@ -132,26 +132,26 @@ You can also add [Dublin Core Metadata Terms](https://www.dublincore.org/specifi
     "https://ruben.verborgh.org/profile/"
   ],
   "dcterms": {
-    "title": "comunica SPARQL endpoint",
-    "creator": "me"
+    "title": "\"comunica SPARQL endpoint\"",
```

Comment:
Since title will always be a string, I don't think this double-quoting is necessary here.